stable/restore/values.yaml

```diff
@@ -56,15 +56,25 @@ admin:
   #   secret: nuodb-client-pem
   #   key: client.pem
 
-database:
-  # name
-  # NuoDB Database name.  must consist of lowercase alphanumeric
-  # characters '[a-z0-9]+'
-  name:
-
 restore:
+
+  ## Provide a name in place of prometheus-operator for `app:` labels
+  ##
+  #nameOverride: ""
+
+  ## Provide a name to substitute for the full names of resources
+  ##
+  #fullnameOverride: ""
+
+  # restore type: [ database | archive ]
+  # default value is "database"
+  #
+  # A database restore restores ALL archives of the database - ie restarts the database at a previous state
+  # An archive restore restores/repairs a single archive in a RUNNING database
+  type: "database"
+
   # name of the existing database being restored 
-  target: ""
+  target: "demo"
```

Comment:
This doesn't correspond to the README.md text. We have default of `{{ .Values.database.name }}` in there.

Reply:
Thanks - fixing...